components/camel-resteasy/src/test/java/org/apache/camel/component/resteasy/test/ResteasyConsumerTest.java

```diff
@@ -0,0 +1,222 @@
+        WebTarget target = client.target(baseUri.toString() + "customer/createCustomer");
+        Response response = target.request(MediaType.APPLICATION_JSON)
+                .post(Entity.entity(customer, MediaType.APPLICATION_JSON_TYPE));
+
+        Assert.assertEquals(200, response.getStatus());
+        return response;
+    }
+
+    private Response deleteCustomer(int id) {
+        Client client = ClientBuilder.newBuilder().build();
+        WebTarget target = client.target(baseUri.toString() + "customer/deleteCustomer?id=" + id);
+        Response response = target.request().delete();
+
+        Assert.assertEquals(200, response.getStatus());
+
+        return response;
+    }
+
+    private Customer getCustomer(int id) {
+        Client client = ClientBuilder.newBuilder().build();
+        WebTarget target = client.target(baseUri.toString() + "customer/getCustomer?id=" + id);
+        Response response = target.request().get();
+
+        Assert.assertEquals(200, response.getStatus());
+
+        return response.readEntity(Customer.class);
+    }
+
+    @Test
+    @InSequence(1)
+    public void testGetAll() throws Exception {
+        String expectedUser1 = "{\"name\":\"Roman\",\"surname\":\"Jakubco\",\"id\":1}";
+        String expectedUser2 = "{\"name\":\"Camel\",\"surname\":\"Rider\",\"id\":2}";
+
+        Client client = ClientBuilder.newBuilder().build();
+        WebTarget target = client.target(baseUri.toString() + "customer/getAll");
+        Response response = target.request().get();
+
+        Assert.assertEquals(200, response.getStatus());
+
+        String users = response.readEntity(String.class);
+        Assert.assertTrue(users.contains(expectedUser1));
+        Assert.assertTrue(users.contains(expectedUser2));
+
+        File file = new File("target/test/consumerTest/all.txt");
+        byte[] encoded = Files.readAllBytes(file.toPath());
+        String responseBody = new String(encoded);
+
+        Assert.assertTrue(responseBody.contains(expectedUser1));
+        Assert.assertTrue(responseBody.contains(expectedUser2));
+    }
+
+    @Test
+    public void testGet() throws Exception {
+        Customer customer = getCustomer(2);
+
+        Assert.assertEquals(new Customer("Camel", "Rider", 2), customer);
+
+        File file = new File("target/test/consumerTest/get.txt");
+        byte[] encoded = Files.readAllBytes(file.toPath());
+        String responseBody = new String(encoded);
+
+        Assert.assertEquals("{\"name\":\"Camel\",\"surname\":\"Rider\",\"id\":2}", responseBody);
+    }
+
+    @Test
+    public void testPost() throws Exception {
+
+        String expectedResponse = "Customer added : Customer{name='TestCreate', surname='TestCreate', id=3}";
+        int customerId = 3;
+
+        Customer customer = new Customer("TestCreate", "TestCreate", customerId);
+        Response response = createCustomer(customer);
+        System.out.println(response.readEntity(String.class));
```

Comment:
Looks like this is from test period debug :)

Reply:
nice catch :)